examples/log/boxfddp_vs_boxddp.log

```diff
@@ -13,78 +13,78 @@ iter 	 cost 	      stop 	    grad 	  xreg 	      ureg 	 step 	 feas
+  39  2.39752e+04  8.93047e+04  2.21174e+03  1.00000e-02  1.00000e-02   0.2500     1
 iter 	 cost 	      stop 	    grad 	  xreg 	      ureg 	 step 	 feas
-  40  2.34988e+04  5.39470e+04  1.65073e+03  1.00000e-03  1.00000e-03   1.0000     1
-  41  2.34988e+04  2.41396e+05  3.60608e+03  1.00000e-02  1.00000e-02   0.0020     1
-  42  2.34774e+04  2.30915e+05  2.02242e+03  1.00000e-02  1.00000e-02   0.0156     1
-  43  2.34235e+04  3.28532e+05  1.21132e+03  1.00000e-02  1.00000e-02   0.0625     1
-  44  2.32739e+04  2.69045e+05  1.03725e+03  1.00000e-02  1.00000e-02   0.2500     1
-  45  2.32719e+04  1.20503e+05  8.29662e+02  1.00000e-02  1.00000e-02   0.0156     1
-  46  2.31589e+04  1.20842e+05  7.41964e+02  1.00000e-02  1.00000e-02   0.2500     1
-  47  2.28254e+04  8.02472e+04  5.70938e+02  1.00000e-03  1.00000e-03   1.0000     1
-  48  2.28254e+04  7.21100e+02  4.23735e+02  1.00000e-02  1.00000e-02   0.0020     1
-  49  2.28250e+04  9.46202e+02  1.81475e+02  1.00000e-01  1.00000e-01   0.0039     1
+  40  2.34988e+04  5.39477e+04  1.65069e+03  1.00000e-03  1.00000e-03   1.0000     1
+  41  2.34988e+04  2.41389e+05  3.60596e+03  1.00000e-02  1.00000e-02   0.0020     1
+  42  2.34774e+04  2.30908e+05  2.02240e+03  1.00000e-02  1.00000e-02   0.0156     1
+  43  2.34235e+04  3.28519e+05  1.21131e+03  1.00000e-02  1.00000e-02   0.0625     1
+  44  2.32739e+04  2.69033e+05  1.03724e+03  1.00000e-02  1.00000e-02   0.2500     1
+  45  2.32719e+04  1.20499e+05  8.29659e+02  1.00000e-02  1.00000e-02   0.0156     1
+  46  2.31588e+04  1.20837e+05  7.41962e+02  1.00000e-02  1.00000e-02   0.2500     1
+  47  2.28254e+04  8.02436e+04  5.70934e+02  1.00000e-03  1.00000e-03   1.0000     1
+  48  2.28254e+04  7.21191e+02  4.23737e+02  1.00000e-02  1.00000e-02   0.0020     1
+  49  2.28250e+04  9.46302e+02  1.81477e+02  1.00000e-01  1.00000e-01   0.0039     1
 *** SOLVE with Box-DDP ***
 iter 	 cost 	      stop 	    grad 	  xreg 	      ureg 	 step 	 feas
    0  2.31599e+07  2.62717e+08  3.86413e+06  1.00000e-09  1.00000e-09   0.5000     1
-   1  2.23361e+07  6.70272e+10  9.81907e+07  1.00000e-09  1.00000e-09   0.0312     1
-   2  1.86846e+07  2.60223e+11  6.80717e+07  1.00000e-09  1.00000e-09   0.0156     1
-   3  1.74778e+07  9.57254e+09  5.50219e+07  1.00000e-09  1.00000e-09   0.1250     1
-   4  1.70684e+07  9.05376e+10  6.05775e+07  1.00000e-08  1.00000e-08   0.0020     1
-   5  1.58771e+07  9.04287e+10  6.03229e+07  1.00000e-07  1.00000e-07   0.0039     1
-   6  1.41584e+07  7.75480e+10  5.34607e+07  1.00000e-06  1.00000e-06   0.0039     1
-   7  1.29359e+07  1.85569e+11  4.29009e+07  1.00000e-05  1.00000e-05   0.0020     1
-   8  1.28508e+07  2.70894e+11  3.79058e+07  1.00000e-04  1.00000e-04   0.0039     1
-   9  1.20118e+07  2.50503e+11  3.44535e+07  1.00000e-04  1.00000e-04   0.0156     1
+   1  2.23353e+07  6.70272e+10  9.81907e+07  1.00000e-09  1.00000e-09   0.0312     1
```

Comment:
This is a big difference in the computation of cost. It should not be here.

Reply:
You should check which variables are used and not initialized with valgrind.

Reply:
Ok, Thanks. I'll have a look

Reply:
This difference might be related to your hardware / setup.
Remember that these files are generated with an specific PC and setup.

Unfortunately, it is hard to make an statement here.

I wonder if we should keep this discussion. I am OK to keep it if @proyan wants to investigate deeper. I am afraid that I don't have time for it.